core/src/main/java/com/amazon/opendistroforelasticsearch/sql/ast/expression/In.java

```diff
@@ -28,7 +28,10 @@
  * Params include the field expression and/or wildcard field expression,
  * nested field expression (@field).
  * And the values that the field is mapped to (@valueList).
+ *
+ * @deprecated use function ("in") instead
  */
+@Deprecated
```

Comment:
I think it's fine to remove this file if the code base has no dependency on it.

Reply:
ok